psi4/driver/driver_findif.py

```diff
@@ -0,0 +1,214 @@
+from __future__ import division
+from psi4.driver.p4util.exceptions import ValidationError
+from psi4 import core
+from itertools import combinations
+import math
+
+
+def displace_cart(mol, geom, salc_list, i_m, step_size):
+    """
+    Given a molecule, a geometry, a list of salcs, an iterable of (SALC displacement index, # steps to displace) tuples,
+    and a step size, name the geometry according to the salc displacements and displace the geometry accordingly.
```

Comment:
going to want to convert these docstrings to numpy-like format for the autodoc tools